src/Microsoft.NetCore.Analyzers/Core/Runtime/TestForEmptyStringsUsingStringLength.cs

```diff
@@ -32,7 +32,7 @@ public sealed class TestForEmptyStringsUsingStringLengthAnalyzer : DiagnosticAna
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Performance,
                                                                              DiagnosticHelpers.DefaultDiagnosticSeverity,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: DiagnosticHelpers.EnabledByDefaultForVsixAndNuget,
```

Comment:
@jinujoseph @kuhlenh There are some rules that we decided to turn on in the VSIX, which were turned off by default even in the NuGet. Do we want them to be turned on by default both for VSIX and NuGet or just the VSIX?

Reply:
You can find these rules by searching for lines deleted in this PR with text `isEnabledByDefault: false`

Reply:
I would think so.

Reply:
I agree, but we should be aware that the NuGet package consumers might now see additional diagnostics from the beta2 package as we are running more analyzers by default, and if they feel otherwise, we need to recommend them to turn these off via ruleset.

Reply:
I mean, as we add new analyzers for other things, consumers of the NuGet packages will also start to see more diagnostics. I don't see a problem with this, as the user is explicitly upgrading the NuGet package. Unlike the VSIX scenario, where they might not initially seem correlated.

Reply:
i think we should enable in nuget as well to be in sync in both vsix and nuget .
And most of them have fix all too